src/Sulu/Bundle/TrashBundle/Application/TrashManager/TrashManager.php

```diff
@@ -102,16 +96,6 @@ public function restore(TrashItemInterface $trashItem, array $restoreFormData =
 
         $translation = $trashItem->getTranslation(null, true);
```

Comment:
Remove `$translation`. Looks like not used after declaration.

Reply:
Fixed!